js/dnd.js

```diff
@@ -137,11 +135,8 @@ module.exports.isMiddle = (domNode, clientX) => {
 module.exports.prepareBookmarkDataFromCompatible = (dataTransfer) => {
   let bookmark = dndData.getDragData(dataTransfer, dragTypes.BOOKMARK)
   if (!bookmark) {
-    const frameProps = dndData.getDragData(dataTransfer, dragTypes.TAB)
-    if (frameProps) {
-      bookmark = siteUtil.getDetailFromFrame(frameProps, siteTags.BOOKMARK)
-      appActions.addSite(bookmark, siteTags.BOOKMARK)
-    }
+    const frame = dndData.getDragData(dataTransfer, dragTypes.TAB)
+    windowActions.onFrameBookmark(frame.get('activeFrameKey'))
```

Comment:
Why I moved this into separate action is because we need to get detail of the frame so we can add it into sites.

Here we need to have access to the window state: https://github.com/brave/browser-laptop/pull/9754/files/e0827f5b092eecfbe915b23838fb9dccc8fdcc5b#diff-be01d0957f991e2400072d82d108450fL142